exchanges/huobi/huobi_test.go

```diff
@@ -281,6 +282,7 @@ func TestFGetAccountInfo(t *testing.T) {
 }
 
 func TestFGetPositionsInfo(t *testing.T) {
+	h.Verbose = true
```

Comment:
rm

Reply:
:D yeah that was just a wip I forgot to remove the review me label